docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -7,39 +7,24 @@
 
 @endsphinxdirective
 
+In case if you are intended to use OpenVINO GPU plugin and offload network inference to Intel® graphics processor, the Intel Graphics Driver should be properly configured on your system.
```

Comment:
```suggestion
If you are intended to use OpenVINO™ GPU plugin and offload inference to Intel® Processor Graphics (GPU), Intel® Graphics Driver must be properly configured on your system.
```

Reply:
This is port of changes from PR #15160 that's already approved by docs team.
Could you please clarify why we have such difference